.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "editor.rulers": [80], // used for .md files
```

Comment:
comments are not allowed in standard JSON but VSC came up with a "jsonc" format:

[JSON with Comments](https://code.visualstudio.com/docs/languages/json#_json-with-comments)
In addition to the default JSON mode following the [JSON specification](https://www.json.org/), VS Code also has a JSON with Comments (jsonc) mode. This mode is used for the VS Code configuration files such as settings.json, tasks.json, or launch.json. When in the JSON with Comments mode, you can use single line (//) as well as block comments (/* */) as used in JavaScript. The mode also accepts trailing commas, but they are discouraged and the editor will display a warning.

this would only be valid for specific filenames. also points to need to have json validation